packages/wrangler/src/index.tsx

```diff
@@ -187,32 +187,52 @@ export async function main(argv: string[]): Promise<void> {
         type: "string",
       });
     },
-    async () => {
+    async (args) => {
+      if ("type" in args) {
+        let message =
+          "The --type option is no longer supported.\n" +
+          "The difference between 'javascript' and 'rust' is automatically inferred.";
+        if (args.type === "webpack") {
+          message +=
+            "\nIf you wish to use webpack then you will need to create a custom build";
+        }
+        console.error(message);
+        return;
+      }
+
       const destination = path.join(process.cwd(), "wrangler.toml");
       if (fs.existsSync(destination)) {
-        console.error(`${destination} already exists.`);
-      } else {
-        const compatibilityDate = new Date().toISOString().substring(0, 10);
-        try {
-          await writeFile(
-            destination,
-            `compatibility_date = "${compatibilityDate}"` + "\n"
-          );
-          console.log(`✨ Succesfully created wrangler.toml`);
-          // TODO: suggest next steps?
-        } catch (err) {
-          console.error(`Failed to create wrangler.toml`);
-          console.error(err);
-          throw err;
+        console.error(`${destination} file already exists!`);
+        const result = await dialogs.confirm(
+          "Do you want to continue initializing this project?"
+        );
+        if (!result) {
+          return;
         }
       }
 
+      const compatibilityDate = new Date().toISOString().substring(0, 10);
+      try {
+        await writeFile(
+          destination,
+          `compatibility_date = "${compatibilityDate}"` + "\n"
+        );
+        console.log(`✨ Succesfully created wrangler.toml`);
+        // TODO: suggest next steps?
```

Comment:
In the future, this might be a great place to suggest a list of "starters" and automatically make a `src` with contents. This may be a good replacement for the `generate` command.